Formula/opencv@2.rb

```diff
@@ -0,0 +1,98 @@
+class OpencvAT2 < Formula
+  desc "Open source computer vision library"
+  homepage "http://opencv.org/"
+  url "https://github.com/opencv/opencv/archive/2.4.13.2.tar.gz"
+  sha256 "4b00c110e6c54943cbbb7cf0d35c5bc148133ab2095ee4aaa0ac0a4f67c58080"
+  revision 1
+  head "https://github.com/opencv/opencv.git", :branch => "2.4"
+
+  option "with-java", "Build with Java support"
+  option "with-tbb", "Enable parallel code in OpenCV using Intel TBB"
+  option "without-numpy", "Use your own numpy instead of Homebrew's numpy"
+  option "without-python", "Build without python support"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "eigen"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "openexr"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+  depends_on "numpy" => :recommended if build.with? "python"
+  depends_on "ffmpeg" => :optional
+  depends_on "tbb" => :optional
+  depends_on :java => :optional
+  depends_on :ant if build.with? "java"
+
+  def arg_switch(opt)
+    (build.with? opt) ? "ON" : "OFF"
+  end
+
+  def install
+    jpeg = Formula["jpeg"]
+
+    args = std_cmake_args + %W[
+      -DCMAKE_OSX_DEPLOYMENT_TARGET=
+      -DBUILD_JASPER=OFF
```

Comment:
Is there a way to say "disable _all_ vendored stuff" so we don't have to play Whac-A-Mole?